public/app/features/dashboard-scene/scene/LibraryVizPanel.tsx

```diff
@@ -106,7 +111,15 @@ export class LibraryVizPanel extends SceneObjectBase<LibraryVizPanelState> {
         });
       }
 
-      this.setState({ panel, _loadedVersion: libPanel.version, isLoaded: true });
+      // extra information from library panel
+      const libraryPanelExtendedInformation = {
+        folderUid: libPanel.folderUid,
+        description: libPanel.description,
+        meta: libPanel.meta,
+        schemaVersion: libPanel.schemaVersion,
+      };
+
+      this.setState({ panel, _loadedVersion: libPanel.version, isLoaded: true, ...libraryPanelExtendedInformation });
```

Comment:
Wondering about this. Why not storing the entire loaded library panel model in the state, rather than this extended information only? In mean - I think i would skip the `_loadedVersion` state property and instead store the `_loadedLibraryPanel: LibraryPanel` in the state and use it as a source for the necessary information when presenting the JSON of an inspected panel, or comparing the loaded version.